templates/deployment.yaml

```
@@ -52,9 +52,12 @@ spec:
           volumeMounts:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- with .Values.extraVolumeDevices }}
+          volumeDevices:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
-
       {{- with .Values.extraVolumes }}
       volumes:
         {{- toYaml . | nindent 8 }}
```

```suggestion
      volumes:
      {{- with .Values.extraVolumes }}
        {{- toYaml . | nindent 8 }}
      {{- end }}
      {{- with .Values.extraVolumeDevices }}
        {{- toYaml . | indent 8 }}
```

i think `extraVolumeDevices` needs to be added to this section as well; this might work?